src/cljs/athens/electron.cljs

```diff
@@ -214,161 +216,122 @@
+       Can be invoked when a db is added, opened, moved, selected or removed.
+       Update local storage value with the new db list"
       (let [current-db-list (:db-picker/all-dbs db)
-            duplicate       (check-duplicate-db
-                              current-db-list
-                              dbpath)]
-        (if (not duplicate)
+            duplicate?       (check-duplicate-db
+                               current-db-list
+                               dbpath)]
+        (if duplicate?
+          {:dispatch [:alert/js (str "Database already in the list")]}
           (let [dbname  (get-db-name dbpath)
                 newdb   {:path dbpath
                          :name dbname}
-                all-dbs (conj
-                          current-db-list
-                          newdb)]
+                all-dbs (conj current-db-list newdb)]
             {:db       (assoc db :db-picker/all-dbs all-dbs)
-             :dispatch [:local-storage/set-db-picker-list]})
-          (js/alert (str "Database already in the list"))))))
+             :dispatch [:local-storage/set-db-picker-list]})))))
+
 
   (reg-event-fx
       :local-storage/set-db-picker-list
       (fn [{:keys [db]} _]
-        "Save current db-picker list to local storage.
-         Why using `pr-str`? Checkout this link for
-         discussion on how to serialize data to store"
+        "Save current db-picker list to local storage. Why using `pr-str`? Checkout
+         this link for discussion on how to serialize data to store"
         ;; https://stackoverflow.com/questions/67821181/how-to-set-and-get-a-vector-in-local-storage
         (let [current-db-list (:db-picker/all-dbs db)]
-          {:local-storage/set! [ "db-picker/all-dbs"
-                                (pr-str current-db-list)]})))
+          {:local-storage/set! ["db-picker/all-dbs" (pr-str current-db-list)]})))
 
   (reg-event-fx
     :local-storage/check-db-picker-list
     (fn [{:keys [db]} _]
       "Check if local storage contains db-picker list.
-       If not it means this is the first time opening
-       Athens or local storage was cleared, if this is
-       the case we update db-picker list with the current
-       running db."
+       If not it means this is the first time opening Athens or local storage was
+       cleared, if this is the case we update db-picker list with the current running db."
       (let [val                 (cljs.reader/read-string
-                                  (js/localStorage.getItem
-                                    "db-picker/all-dbs"))
+                                  (js/localStorage.getItem "db-picker/all-dbs"))
             current-db-filepath (:db/filepath db)]
         (cond
-          (nil? val)  {:dispatch [:db-picker/add-new-db
-                                  current-db-filepath]}
+          (nil? val)  {:dispatch [:db-picker/add-new-db current-db-filepath]}
           :else       {:db (assoc db :db-picker/all-dbs val)}))))
```

Comment:
Just `if` here, no need for `cond`

Reply:
Ok will fix it, after using if I felt it hard to read so went with `cond`